### DISCUSSION TITLE
2021-06-23 Weekly Meeting 9 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#51] [Weekly Meetings]
Weekly Meeting 9 is happening on Wednesday 2021-06-23 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- [W3C Community Group Charter Ratification](https://github.com/omigroup/OMI/pull/45)
- [OMI Community Virtual Space Design](https://github.com/omigroup/OMI/issues/44)
- OMI OpenCollective
  - [About OpenCollective](https://opencollective.com/opensource)
  - Discussion about OpenCollective going on in the #omi-community-development channel
- [OMI Homepage / Documentation Site](https://github.com/omigroup/OMI/issues/49)
- Call for Social Media / Marketing volunteers
  - DM Evo, Jesse, or Robert on Discord if interested
- OMI Happy Hour 6/25 3:00 PM PDT

**Comment by mrmetaverse** [comment]
## OMI meeting notes
We started a note on [hackmd](https://hackmd.io/@mrmetaverse/omi-062321) :link:

**Reply by robertlong** [reply]
Final notes available here: https://hackmd.io/@robertlong/omi-meeting-9
